dataland-frontend/src/components/forms/CreateEuTaxonomyForFinancials.vue

```diff
@@ -516,6 +519,37 @@ export default defineComponent({
         .catch((e) => console.log(e));
     },
 
+    /**
+     * Parses received financial service types and eligibility KPIs data
+     * @param receivedFormInputsModelData Received data
+     * @returns An object with financial service types and eligibility KPIs which can be directly merged into the formInputsModel, which will then populate expected KPI section fields
+     */
+    extractEligibilityKpis(receivedFormInputsModelData: ObjectType) {
+      const kpiKeys = Object.keys(euTaxonomyKPIsModel.kpisFieldNameToFinancialServiceType);
+      return kpiKeys
+        .map((key: string) => {
```

Comment:
`financialServiceTypeKey` for consistency then, maybe? Or what is the benefit of shorter names?

Reply:
Code readability only. I'll change it for consistency.

Reply:
Actually, I think it will be better to leave it as `key`. We already know it is coming from `financialServiceTypeKeys`